web/src/components/MalteHelpForm.tsx

```diff
@@ -0,0 +1,242 @@
+
+import { SecurityIcon, SupportIcon } from '../assets'
+import Icon from '../components/base/Icon'
+import { Container } from './Feedback'
+import Input from './base/Input'
+import InputSection from './base/InputSection'
+import RadioElement, { RadioGroup } from './base/RadioElement'
+import TextButton from './base/TextButton'
+
+const Note = styled.div`
+  display: flex;
+  padding-bottom: 10px;
+`
+
+const StyledIcon = styled(Icon)<{ direction: 'ltr' | 'rtl' }>`
+  width: 24px;
+  height: 24px;
+  ${props => (props.direction === 'ltr' ? 'padding-right: 20px' : 'padding-left: 20px')};
+  flex-shrink: 0;
+`
+
+const Divider = styled.hr`
+  margin: 12px 0;
+  background-color: ${props => props.theme.colors.textSecondaryColor};
+  height: 1px;
+  border: none;
+`
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`
+
+const SubmitErrorHeading = styled.h5`
+  margin: 0;
+  font-size: ${props => props.theme.fonts.subTitleFontSize};
+`
+
+const ErrorSendingStatus = styled.div`
+  background-color: ${props => props.theme.colors.invalidInput}35;
+  padding: 20px 10px;
+  margin: 10px 0;
+`
+
+type ContactChannel = 'email' | 'telephone' | 'person'
+type ContactGender = 'male' | 'female' | 'any'
+type SendingStatusType = 'idle' | 'sending' | 'failed' | 'successful'
+
+type MalteHelpFormProps = {
+  cityCode: string
+  languageCode: string
+  helpButtonOffer: OfferModel
+}
+
+const MalteHelpForm = ({ languageCode, cityCode, helpButtonOffer }: MalteHelpFormProps): ReactElement => {
+  const { t } = useTranslation('malteHelpForm')
+  const [sendingStatus, setSendingStatus] = useState<SendingStatusType>('idle')
+  const [submitted, setSubmitted] = useState(false)
+  const [contactChannel, setContactChannel] = useState<ContactChannel>('email')
+  const [email, setEmail] = useState('')
+  const [telephone, setTelephone] = useState('')
+  const [name, setName] = useState('')
+  const [roomNumber, setRoomNumber] = useState('')
+  const [contactType, setContactType] = useState<ContactGender>('any')
+  const [comment, setComment] = useState('')
+  const missingData =
+    !name.length ||
+    (!email.length && contactChannel === 'email') ||
+    (!telephone.length && contactChannel === 'telephone')
+  const dashboardRoute = cityContentPath({ languageCode, cityCode })
+
+  const submitHandler = useCallback(
+    (event: SyntheticEvent<HTMLFormElement>) => {
+      const form = event.currentTarget
+      if (!form.checkValidity()) {
+        // event.preventDefault()
+        event.stopPropagation()
+        const invalidInput = form.querySelector(':invalid:not(fieldset)')
+        invalidInput?.scrollIntoView({ behavior: 'smooth' })
+        setSendingStatus('idle')
+      } else {
+        setSendingStatus('sending')
+        const request = async () => {
+          await submitHelpForm({ cityCode, languageCode, helpButtonOffer })
+          setSendingStatus('successful')
+        }
+
+        request().catch(err => {
```

Comment:
Please avoid mixing async/await and then/catch

Reply:
Where is it mixed? 😅 I used the same logic as in the feedback container.